README.md

```diff
@@ -6,4 +6,40 @@
 [![Stackage Nightly](http://stackage.org/package/hintman/badge/nightly)](http://stackage.org/nightly/package/hintman)
 [![Build status](https://secure.travis-ci.org/kowainik/hintman.svg)](https://travis-ci.org/kowainik/hintman)
 
-GitHub application to suggest hints
\ No newline at end of file
+GitHub application to suggest hints
+
+
+## How to test GitHub application?
+
+### Prerequisites
+
+1. `smee`:
+   + `npm install --global smee-client`
+2. `cabal` or `stack`
+
+### How to run
+
+1. Run `smee` forwarder in a separate terminal window:
+    + `smee -u "https://smee.io/uTG0BCXnjq4DEff7" -p 8080`
+2. Run `backend`:
+    + `cabal new-run hintman -- -p 8080`
+3. Open GitHub application and press `Configure` button to add it to your account:
+    + https://github.com/apps/hint-man
+4. After adding access to your personal profie, choose `Select repositories`
+   repository access and it this application to any of your repositories. You
+   adding and saving you should be able to see the following:
```

Comment:
You addidng and saving?

Reply:
Yeah, need to reword this sentence. I don't know what happened with it.